content/master/cli/command-reference.md

```diff
@@ -828,7 +828,8 @@ Configuration/platform-ref-aws                             v0.9.0    True
 
 The `crossplane beta validate` command validates 
 [compositions]({{<ref "../concepts/compositions">}}) against provider or XRD 
-schemas using the Kubernetes API server's validation library.
+schemas using the Kubernetes API server's validation library
+with additional validation checks such as Unknown Field Check.
```

Comment:
I don't think Unknown Field Check is a proper noun, so no need to capitalize it. How about this suggestion that may flow more naturally?
```suggestion
with additional validation such as checking for unknown fields, a common source of difficult to debug issues in Crossplane.
```